.github/workflows/test-release-version-check.yml

```diff
@@ -1,24 +1,106 @@
 ---
-
 name: Test Version Check Action
 
 on:
   pull_request:
-    path:
+    paths:
       - "release-check-version"
   push:
     branches:
       - "master"
   workflow_dispatch:
-    inputs:
-      version_number:
-        description: "New Version"
-        required: true
+    inputs: {}
```

Comment:
This line can actually be deleted.  I know we use it in our other workflows, but I wanted to call it out in case we wanted to remove it going forward.
```suggestion
```